siliconcompiler/remote/client.py

```diff
@@ -750,10 +750,12 @@ def success_action(resp):
 
     # Print server-side version info.
     version_info = response_info['versions']
-    chip.logger.info('Software version info:')
-    chip.logger.info(f'  Server version             : {version_info["sc_server"]}')
-    chip.logger.info(f'  Server\'s SC version        : {version_info["sc"]}')
-    chip.logger.info(f'  Server\'s SC Schema version : {version_info["sc_schema"]}')
+    version_suffix = ' version'
+    max_version_string = max([len(s) for s in version_info.keys()]) + len(version_suffix)
+    chip.logger.info('Server software versions:')
+    for name, version in version_info.items():
+        print_name = f'{name}{version_suffix}'
+        print(f'  {print_name: <{max_version_string}}: {version}')
```

Comment:
Probably better to first completly build the string and then output it with `chip.logger.info()`. Otherwise this won't obey logging levels and all the other things.